packages/material-ui/src/Typography/Typography.test.js

```diff
@@ -50,17 +44,30 @@ describe('<Typography />', () => {
 
     expect(container.firstChild).to.have.class(classes.alignCenter);
   });
-  ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'subtitle1', 'body2', 'body1', 'caption', 'button'].forEach(
-    (variant) => {
-      it(`should render ${variant} text`, () => {
-        // @ts-ignore literal/tuple type widening
-        const { container } = render(<Typography variant={variant}>Hello</Typography>);
-
-        expect(classes[variant] != null).to.equal(true);
-        expect(container.firstChild).to.have.class(classes[variant]);
-      });
-    },
-  );
+  [
+    'h1',
+    'h2',
+    'h3',
+    'h4',
+    'h5',
+    'h6',
+    'subtitle1',
+    'body2',
+    'body1',
+    'caption',
+    'button',
+    'overline',
+  ].forEach((variant) => {
+    it(`should render ${variant} text`, () => {
+      // @ts-ignore literal/tuple type widening
+      const { container } = render(<Typography variant={variant}>Hello</Typography>);
+
+      // @ts-ignore
+      expect(classes[variant] != null).to.equal(true);
```

Comment:
Could you clarify what exactly you're expecting here? 
- Should `classes` not have a property called `variant`? => 
  ```suggestion
      expect(classes).not.to.have.property(variant);
  ```
- Should `classes` have a property called `variant` that is undefined? => 
  ```suggestion
      expect(classes).to.have.property(variant, undefined);
  ```
- Should `classes` have a property called `variant` that is null? => 
  ```suggestion
      expect(classes).to.have.property(variant, null);
  ```

It's not necessary to use any of the suggestions but at least there should be an explanation why we use the weak comparison and in which case either applies.

Reply:
Improved. This is the final check: 

`expect(classes).to.have.property(variant);`